src/api/flaskr/service/learn/runscript_v2.py

```
@@ -138,87 +151,217 @@ def run_script(
-        finally:
-            lock.release()
         return
     else:
         app.logger.warning("lockfail")
         yield make_script_dto("text_end", "", None)
     return
+
+
+def get_run_status(
+    app: Flask,
+    shifu_bid: str,
```

Ruff (0.14.1)

Unused function argument: `shifu_bid`

(ARG001)